src/nvim/api/vim.c

```diff
@@ -545,6 +545,17 @@ Object nvim_exec_lua(String code, Array args, Error *err)
   return nlua_exec(code, args, err);
 }
 
+Object nvim_notify(String msg, Integer log_level, Dictionary opts, Error *err)
+  FUNC_API_SINCE(7)
+{
+  Array args = ARRAY_DICT_INIT;
```

Comment:
looks like a leak to me, maybe `FIXED_TEMP_ARRAY` ?

Reply:
indeed :+1:  fixed